src/stream/stream_backend.rs

```diff
@@ -265,7 +266,7 @@ fn create_rtsp_stream(
                 format!(
                     concat!(
                         "v4l2src device={device}",
-                        " ! video/x-h264,width={width},height={height},framerate={interval_denominator}/{interval_numerator}",
+                        " ! video/x-h264,width={width},height={height},framerate={interval_denominator}/{interval_numerator},type=video",
```

Comment:
This should be squashed in the original commit, also this specific commit should be one of the first, since it adds the server source file and the base of the functionality, please correct the commit order.